src/tools/tests/goldens/WriterReaderExample.kt

```diff
@@ -16,7 +16,12 @@ object IngestionPlan : Plan(
             "Writer",
             "",
             mapOf(
-                "data" to HandleConnection(StorageKeyParser.parse(""), HandleMode.Write, null, null)
+                "data" to HandleConnection(
+                    StorageKeyParser.parse(""),
```

Comment:
This really needs to be CreatableStorageKey("nameInRecipe"), if not this PR, in the next one.

What I mean is, if you have
recipe Foo 
   bar: create

Then you need to have CreateableStorageKey("bar") in HandleConnections that use it.

Reply:
Just commented about this in #4894, we're (I'm) going to impl this in the next PR. 
@mmandlis